packages/extension-sample/src/setup-app.ts

```diff
@@ -26,7 +26,7 @@ class Sample extends ApplicationExtension {
         return [
             {
                 exact: true,
-                path: this.getConfig()?.path || defaultPath,
+                path: this.getConfig().path || defaultPath,
```

Comment:
I noticed that the path value must start with a '/' followed by a string. Is that intentional? 

Right now, if the user uses other valid URLs, the path will not work. 

For example, other valid URLs could be:
'foo-page'
'foo-page/bar'
'../foo-page'

Reply:
That's the expected `path` syntax by react-router: https://v5.reactrouter.com/web/api/Route/path-string-string.

Reply:
I see, you're right. It would be nice to document that to make it more obvious